week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,9 +58,21 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
+  let finalArr = [];
+  for (i in arr) {
```

Comment:
```suggestion
  for (let i = 0; i < arr.length; i++) {
```
Also worth remembering that the more standard way of writing a `for` loop over an array is like above. But your method also works too 👍